### DISCUSSION TITLE
Finite rate reaction chemistry and INITIAL_UNMIXED_FRACTION

### DISCUSSION THREAD

**Discussion by vijay00901** [firemodels/fds#13419] [Q&A] [answered]
I simulated a backdraft phenomenon using LES and a finite rate chemistry in FDS with the default INITIAL_UNMIXED_FRACTION value of 1.  Since the chemistry is finite rate, does changing the initial unmixed fraction alter the output parameters such as heat release or does the initial unmix fraction has any influence in a finite rate chemistry reaction? Please let me know. Thank you.

**Comment by drjfloyd** [comment, marked as answer]
Per the Technical Reference Guide the combustion model process is to first perform the mixing calculation in the gas cell and then integrate the combustion reactions over the time step using the mixed gas concentrations as the initial value. If you change the unmixed fraction, you will likely change the composition of the mixed state. The exception would be in any gas cells where the mixing calculation results in complete mixing.

**Reply by vijay00901** [reply]
Thank you so much Dr. Floyd
